elucidation-bundle/pom.xml

```diff
@@ -40,9 +40,9 @@
         <!-- Elucidation dependencies -->
 
         <dependency>
-            <groupId>com.fortitudetec</groupId>
+            <groupId>org.kiwiproject</groupId>
             <artifactId>elucidation-common</artifactId>
-            <version>${elucidation-common.version}</version>
+            <version>4.1.3-SNAPSHOT</version>
```

Comment:
Why hard coding this? Especially since you left the one above on line 29 alone...